react-app/src/stylesheets/Page.scss

```diff
@@ -0,0 +1,47 @@
+@import './_variables.scss';
+/* .page {
+    background-color: white;
+    background-size: cover;
+    height: 100%;
+} */
+
+.row {
+  flex-wrap: nowrap;
+}
+.col-xl-3,
+.col-lg-3,
+.col-md-3,
+.col-sm-3,
+.col-3 {
+  padding-left: 0px;
+  padding-right: 0px;
+}
```

Comment:
DP1 -  I think there is a `no-gutters` className in bootstrap that we could be using here instead of overriding the .col-*-3 padding.

Reply:
This is a pretty nifty cheatsheet that I often use when in react-bootstrap land: https://hackerthemes.com/bootstrap-cheatsheet/#no-gutters